lib/browser/browser.js

```diff
@@ -73,7 +73,9 @@ module.exports = class Browser {
             sessionId,
             ...sessionOpts,
             ...detectedSessionEnvFlags,
-            ...this._config.sessionEnvFlags
+            ...this._config.sessionEnvFlags,
+            options: sessionOpts,
```

Comment:
In new wdio version attach to existing browser with `devtools` protocol works without any problem when you pass instance of browser created using `remote` method. Like this:
```
const client = await remote(opts);
const client2 = await attach(client);
```

But it does not work when you pass instance from one process to another. I get error `route not found`. Looking to wdio code I found that `options` should be passed explicitly - https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/index.ts#L85.